lib/result.rb

```diff
@@ -0,0 +1,17 @@
+def result(your_move, cpu_move)
```

Comment:
Good method vs CPU, clear and easy to understand. Would be interesting to see how this would be implemented in a two player game